kratos/python_scripts/python_solver.py

```diff
@@ -162,22 +162,22 @@ def _ImportModelPart(self, model_part, model_part_import_settings):
             if (model_part_import_settings.Has("reorder") and model_part_import_settings["reorder"].GetBool()):
                 tmp = KratosMultiphysics.Parameters("{}")
                 KratosMultiphysics.ReorderAndOptimizeModelPartProcess(model_part, tmp).Execute()
-            self.print_on_rank_zero("::[PythonSolver]::", "Finished reading model part from mdpa file.")
+            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Finished reading model part from mdpa file.")
 
         elif (input_type == "rest"):
-            self.print_on_rank_zero("::[PythonSolver]::", "Loading model part from restart file.")
+            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Loading model part from restart file.")
             from restart_utility import RestartUtility
             RestartUtility(model_part, self._GetRestartSettings(model_part_import_settings)).LoadRestart()
-            self.print_on_rank_zero("::[PythonSolver]::", "Finished loading model part from restart file.")
+            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Finished loading model part from restart file.")
 
         elif(input_type == "use_input_model_part"):
             pass
 
         else:
             raise Exception("Other model part input options are not yet implemented.")
 
-        self.print_on_rank_zero("ModelPart", model_part)
-        self.print_on_rank_zero("::[PythonSolver]:: ", "Finished reading model part.")
+        KratosMultiphysics.Logger.PrintInfo("ModelPart", model_part)
```

Comment:
Could we make it so that this is only printed if the echo level is larger than 0? I find it a bit annoying that it is always printed, specially if one is running many simulations.

Reply:
Not in this PR, this only cleans up a bit

You can open an issue to discuss it, this affects many ppl
Personally I would leave it as is, I usually check the modelpart I am using.

I can understand though that it might be verbose in your case
I think a better solution for you would be to increase the logger lvl, have you tried that?